app/templates/admin/permissions/event-roles.hbs

```diff
@@ -17,18 +17,19 @@
     </tr>
   </thead>
   <tbody>
-    {{#each model.roles as |item|}}
+    {{#each model.roles as |role|}}
       <tr>
-        <td>{{item}}</td>
-        {{#each (range 1 6) as |number|}}
+        <td>{{role}}</td>
+        {{#each (range 1 6) as |index|}}
```

Comment:
@CosmicCoder96 please understand the difference b/w a component and a helper :) ... You're talking about a component now.